### ISSUE TITLE
Notify user if they are submitting a resource that has already been added

### ISSUE THREAD

**Issue by dajbelshaw** [toolness/weblitmapper#29]
At the moment I think the WebLitMapper does nothing and dumps you back in the gallery, which is confusing for the user.

(surfaced from https://etherpad.mozilla.org/weblitmapper-feedback)

**Comment by toolness** [comment]
Hmm, so it should actually present the user with the existing resource info--e.g. its name, description, and current tags--and allow the user to modify them if they wish. Any changes they make should be applied to the existing resource, so e.g. if they change the description, it'll be changed on the original resource. In other words, it's identical to clicking the "edit" button on a resource in the gallery.

That said, we're not currently doing any normalization on the URLs (see #22), so some resources that are semantically identical will be recognized by the app as completely different (e.g. the en-US and en-GB localizations of a site).

**Comment by toolness** [comment]
I guess if the user is coming from the bookmarklet, we could put an alert at the top of the page saying "this resource is already mapped, but you can make any changes here if you like." Would that help?

**Comment by dajbelshaw** [comment]
Yes, anything to make it super-obvious. Especially given that the first thing that someone submits is likely to already be in there!
